homeassistant/components/__init__.py

```diff
@@ -21,6 +22,15 @@ def is_on(hass: HomeAssistant, entity_id: str | None = None) -> bool:
 
     If there is no entity id given we will check all.
     """
+    report(
+        (
+            "uses homeassistant.components.is_on."
+            " This is deprecated and will stop working in Home Assistant 2024.6, it"
+            " should be updated to use the function of the platform directly."
+        ),
+        error_if_core=False,
```

Comment:
If it's not currently used in core then maybe this can be set to true?

Reply:
Makes totally senses, thanks